pkg/network/bridge/plugin.go

```diff
@@ -80,10 +84,16 @@ func bridgeExists(bridge string) bool {
 	}
 }
 
-func createFakeDevice() {
-	glog.V(3).Info("Creating fake block device")
-	cmd := exec.Command("mknod", fakeDevicePath, "b", "1", "1")
-	cmd.Run()
+func createFakeDevice() error {
+	if _, err := os.Stat(fakeDevicePath); err == nil {
```

Comment:
Extra: it might be worth checking os.IsNotExist(err); otherwise we'd warn about device already existing even if stat failed for other reason.